### PR TITLE
My 29195 rate limiting on track traces and delivery options

### PR DESCRIPTION
**Pull request by RichardPerdaan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -104,6 +104,13 @@ <h3 id="1_C_3">1.C.3 DELETE</h3>
                     DELETE is used to delete an existing object (piece of data). This method always returns HTTP/1.1
                     204 No Content unless otherwise specified.
                 </p>
+                <h2 id="1_D">1.D Rate limiting</h2>
+                <p>
+                    Rate limits apply to many of our endpoints. You cannot receive data after you have exceeded
+                    the limit. The rate limiting will be reached when you make more than 100 requests per minute
```
Comment (EdieLemoine) [inline index.html]:
```suggestion
                    the limit. The rate limit will be reached when you make more than 100 requests per minute.

```

File: index.html [hunk 2]
```diff
@@ -104,6 +104,13 @@ <h3 id="1_C_3">1.C.3 DELETE</h3>
                     DELETE is used to delete an existing object (piece of data). This method always returns HTTP/1.1
                     204 No Content unless otherwise specified.
                 </p>
+                <h2 id="1_D">1.D Rate limiting</h2>
+                <p>
+                    Rate limits apply to many of our endpoints. You cannot receive data after you have exceeded
+                    the limit. The rate limiting will be reached when you make more than 100 requests per minute
+                    The limit automatically resets after 1 minutes.<br><br>
```
Comment (EdieLemoine) [inline index.html]:
```suggestion
                    The limit automatically resets after 1 minute.<br><br>

```

File: index.html [hunk 3]
```diff
@@ -104,6 +104,13 @@ <h3 id="1_C_3">1.C.3 DELETE</h3>
                     DELETE is used to delete an existing object (piece of data). This method always returns HTTP/1.1
                     204 No Content unless otherwise specified.
                 </p>
+                <h2 id="1_D">1.D Rate limiting</h2>
+                <p>
+                    Rate limits apply to many of our endpoints. You cannot receive data after you have exceeded
+                    the limit. The rate limiting will be reached when you make more than 100 requests per minute
+                    The limit automatically resets after 1 minutes.<br><br>
+                    Example: The limit is 100 requests per 1 minutes. If you reach 100 in 15 seconds, you will be able to continue 45 seconds later.
```
Comment (EdieLemoine) [inline index.html]:
```suggestion
                    Example: If you make 100 requests to the same endpoint within 15 seconds, you will be able to continue 45 seconds later.

```

File: index.html [hunk 4]
```diff
@@ -4882,6 +4896,10 @@ <h3 id="7_U_2">7.U.2 Example</h3>
 </pre>
                 <h1 id="8">8. Delivery options</h1>
                 <h2 id="8_A">8.A Get delivery options</h2>
+                <p>
+                    A rate limiting has been applied to the delivery options endpoint.
```
Comment (EdieLemoine) [inline index.html]:
```suggestion
                    This endpoint has a rate limit.

```

**Review by EdieLemoine — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (RichardPerdaan) [reply index.html]: deze had ik al gedaan

**Review by EdieLemoine — APPROVED** [review]

**Review by joerivanveen — APPROVED** [review]

**Pull request merged**